src/main/webapp/i18n/de/notification.json

```diff
@@ -77,8 +77,8 @@
                 "programmingTestCasesChanged": "Die Testfälle der Programmieraufgabe \"{{ placeholderValues.1 }}\" im Kurs \"{{ placeholderValues.0 }}\" wurden aktualisiert. Die Abgaben der Studenten sollten erneut gebaut und getestet werden, um neue Ergebnisse zu erstellen.",
                 "newManualFeedbackRequest": "Die Programmieraufgabe \"{{ placeholderValues.1 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat eine neue manuelle Feedbackanfrage. Bitte bewerte das Feedback vor der Deadline.",
                 "illegalSubmission": "Die Aufgabe \"{{ placeholderValues.1 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat illegale Abgaben von Studenten.",
-                "newExercisePost": "Die Aufgabe \"{{ placeholderValues.8 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat einen neuen Beitrag: \"{{ placeholderValues.2 }}\"",
-                "newLecturePost": "Die Vorlesung \"{{ placeholderValues.8 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat einen neuen Beitrag: \"{{ placeholderValues.2 }}\".",
+                "newExercisePost": "Die Aufgabe \"{{ placeholderValues.5 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat einen neuen Beitrag: \"{{ placeholderValues.2 }}\"",
+                "newLecturePost": "Die Vorlesung \"{{ placeholderValues.5 }}\" im Kurs \"{{ placeholderValues.0 }}\" hat einen neuen Beitrag: \"{{ placeholderValues.2 }}\".",
```

Comment:
Can you maybe explain that with a comment? Why do we access `placeholderValues.5` instead of some real key? 🤔

Reply:
These are the texts for notifications. A notification does not contain the complete exercise or lecture object for example, but just an array of strings with placeholders. They are accessed based on an index.
There are lots of different types of notifications and they can have a lot of different placeholders. So if you would want an object with named attributes instead of a string array it would have to cover every possible placeholder value and most of them would be null. So we decided to use a string array. But this design decision was also not part of this PR.
I just noticed that we set the wrong index for these two notification texts so I fixed them in addition to the incorrect mail subject.